### PR TITLE
Added PHPixie 3.2

### PR DESCRIPTION
**Pull request by dracony**

### CODE REVIEW

File: phpixie-3.2/_benchmark/hello_world.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+url="$base/$fw/web/index.php"
```
Comment (kenjis) [inline phpixie-3.2/_benchmark/hello_world.sh]: Would you please change the URL like `"$base/$fw/web/index.php/hello/index"`?

**Comment by dracony** [comment]
Done

**Comment by kenjis** [comment]
Sorry, I hope you add the routing definition for `hello/index` and call kind of Hello controller.
I don't understand how to set routing when I see your code.

**Comment by dracony** [comment]
PHPixie doesn't really have the conventional router to controller workflow. It works sort of like middleware where each processor can either handle the request himself or send it somewhere else. Since the hello world app does not really require a dispatcher I did not create one.

The router still kicks in though (its in the /assets/config/http.php file), it specifies which bundle to send the request to.  It is set to send everything to the 'app' bundle

**Comment by kenjis** [comment]
How do I do when I add another route (another page)?

**Comment by dracony** [comment]
Well you'd have some url parameter like: https://github.com/PHPixie/Demo-Todo/blob/master/bundles/app/assets/config/routeResolver.php#L16

and add a dispatching processor that takes that parameter into account like so: https://github.com/PHPixie/Demo-Todo/blob/master/bundles/app/src/Project/App/HTTPProcessor.php

And then you can nest them even deeper.e.g having <processor>/<subprocessor>/<action>/<passtootherprocessor> URLs

**Comment by kenjis** [comment]
Thank you for your explanation.

> Since the hello world app does not really require a dispatcher I did not create one.

To merge this PR, I need  a router and a dispatcher. Because the goal of this project is not measuring hello world application. Please see:

* https://github.com/kenjis/php-framework-benchmark#php-framework-benchmark
* https://github.com/kenjis/php-framework-benchmark#benchmarking-policy

All other frameworks have the route for the URL.

**Comment by dracony** [comment]
I changed it a bit to look more conventional. The way it is now if you try to access any URL other than '/web/index.php/hello' you will get a 404 error.

**Comment by kenjis** [comment]
Thank you! That is exactly what I want.

**Comment by kenjis** [comment]
Updated the result: https://github.com/kenjis/php-framework-benchmark#hello-world-benchmark

**Comment by dracony** [comment]
Thanks. It does perform waaaaaay better on my vagrant setup though. Are you using vagrant/Virtualbox/etc or a real server?

**Comment by kenjis** [comment]
See https://github.com/kenjis/php-framework-benchmark#benchmarking-environment
My Host is MBA Mid 2013.

**Pull request merged**